includes/Admin/SetupWizard.php

```diff
@@ -64,11 +64,9 @@ public function set_user_cap( $caps ) {
      * @return void
      */
     public function enqueue_scripts() {
-        $suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
-
         if ( ! is_admin() ) {
-            wp_register_script( 'selectWoo', WC()->plugin_url() . '/assets/js/selectWoo/selectWoo.full' . $suffix . '.js', [ 'jquery' ], '1.0.1', true );
-            wp_register_script( 'wc-enhanced-select', WC()->plugin_url() . '/assets/js/admin/wc-enhanced-select' . $suffix . '.js', [ 'jquery', 'selectWoo' ], WC_VERSION, true );
+            wp_register_script( 'selectWoo', WC()->plugin_url() . '/assets/js/selectWoo/selectWoo.full.js', [ 'jquery' ], '1.0.1', true );
```

Comment:
the previous implementation was ok. kindly revert previous code for selectWoo and wc-enhanced-select files